clash-vexriscv/src/ffi/impl.cpp

```diff
@@ -6,6 +6,31 @@
 #include "verilated.h"
 #include "interface.h"
 
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <string.h>
+#include <arpa/inet.h>
+#include <fcntl.h>
+#include <sys/ioctl.h>
+#include <netinet/tcp.h>
+
+typedef struct {
+	int server_socket, client_handle;
+	struct sockaddr_in server_addr;
+	struct sockaddr_storage server_storage;
+
+	uint32_t timer;
+	socklen_t addr_size;
+	uint32_t self_sleep;
+	uint32_t check_new_connections_timer;
+	uint8_t rx_buffer[100];
+	int32_t rx_buffer_size;
+	int32_t rx_buffer_remaining;
+	JTAG_INPUT prev_input;
+
+	uint64_t tck_change_counter;
```

Comment:
We had this in here for debugging purposes, I doesn't seem like it's used anywhere. Should we remove it?